src/pages/SubmissionDetails.js

```diff
@@ -116,9 +116,8 @@ export const SubmissionDetails = ({ submissionId }) => {
     }
 
     loadRunsData()
-    loadRunSetData().then(runSet => {
-      runSet && loadMethodsData(runSet.method_version_id)
-    })
+      .then(() => loadRunSetData())
+      .then(runSet => runSet && loadMethodsData(runSet.method_version_id))
```

Comment:
TOL: I am wondering if there is a `.catch()` that we can use here to catch the errors from the previous failed promise and display them? Or would the `try..catch` inside individual methods be enough for that?

Reply:
good question... I *think* the try/catch inside individual methods are enough

Reply:
Talked face-to-face and the `try...catch` should be sufficient.